lib/generators/templates/tasks/development.rake

```diff
@@ -0,0 +1,11 @@
+namespace :development do
+  desc "Loads seed data into development."
+  task seed: :environment do
+    Seeder.load_seeds
```

Comment:
If we don't care about unit testing this, we could always load the seeds directly in this file, like before.

Curious to hear what others think.